apps/src/music/views/MusicView.jsx

```diff
@@ -308,6 +345,21 @@ class UnconnectedMusicView extends React.Component {
     }
   };
 
+  loadLevelData = async () => {
```

Comment:
couple thoughts/questions on this:
- Will this work on a `/levels/:level_id` page?
- We will need this to also work eventually on a `/projects/musiclab/:channel_id` page, this probably won't work for that
- We should also probably move this logic out of `MusicView`, at least into a helper that can be shared across labs

Reply:
In general, I worry about using the current path to determine the request url. I think this would work better if we had `level_id` and `script_id` stored somewhere and used those to construct the url. We'll also need to support a null `script_id` for the cases listed above. I'll also need these for the projects work, I'll think about this more today

Reply:
All great points.  Made some progress, though of course open to more feedback:

- As noted, I've added the route for a level page.  
- I also updated the loader code to use values from existing redux stores where available, and used the new `labRedux` store for one value (a level page's level ID) that wasn't already available anywhere in our redux stores.  Specifically, the URL generation for a script level or level `level_data` call now uses these values, rather than borrowing from the existing URL.  
- I also moved the various loading functions into a standalone utility file, though open to moving them to somewhere even more shareable.

Reply:
> * I also updated the loader code to use values from existing redux stores where available, and used the new `labRedux` store for one value (a level page's level ID) that wasn't already available anywhere in our redux stores.  With this, the URL generation for a script level or level `level_data` call now uses these values, rather than borrowing from the existing URL.

Update: a level page's ID is now stored in `progressRedux`, rather than in `labRedux`.